src/script/ismine.h

```diff
@@ -22,7 +22,9 @@ enum isminetype
     ISMINE_WATCH_SOLVABLE = 2,
     ISMINE_WATCH_ONLY = ISMINE_WATCH_SOLVABLE | ISMINE_WATCH_UNSOLVABLE,
     ISMINE_SPENDABLE = 4,
```

Comment:
I'm very concerned about reusing the old name to mean the UA use case, because the new meaning will conflict with any upstream backports we make, enabling this problem to silently re-occur. Given that we need to rename some places anyway to use `ISMINE_SPENDABLE_ANY` where we explicitly want to handle both, I would prefer that we either keep `ISMINE_SPENDABLE` being the legacy selector, and use e.g. `ISMINE_ACCOUNT_SPENDABLE` for the UA case, or rename both.

Reply:
I agree that this is a tricky point, because there's a distinction now that we didn't have before, where sometimes values of this type are used as filters, and sometimes values of this type are used as flag sets.

Perhaps the better way to go about this would be to create two separate types: `ismineflags` and `isminefilter` to distinguish the uses? Then the compiler would help us catch any future incompatibilities.